modules/i18n/src/translations/en-US/portals/remote-user-stores.ts

```diff
@@ -0,0 +1,245 @@
+        }
+    },
+    pages: {
+        create: {
+            backButton: "Go back to user stores",
+            description: "Onboard the users in your remote user store to {{ productName }}.",
+            message: {
+                classic:
+                    "If your requirement is only for authentication, we recommend using the <1>Optimized User Store Connection</1> for efficiency.",
+                optimized:
+                    "This configuration supports Authentication Only. User and group management features are not available in this setup. If user management is an essential requirement, please use the <1>Classic User Store Connection</1> instead."
+            },
+            notifications: {
+                createUserStore: {
+                    genericError: {
+                        description: "There was an error while creating the user store.",
+                        message: "Something went wrong!"
+                    },
+                    success: {
+                        description: "The user store has been added successfully!",
+                        message: "User store added successfully!"
+                    }
+                }
+            },
+            stepper: {
+                step1: {
+                    description: "Provide the basic details to identify and connect your user store.",
+                    title: "General Details"
+                },
+                step2: {
+                    description:
+                        "Complete the required settings to integrate your connected user store, enabling smooth user access to applications.",
+                    title: "Configurations"
+                }
+            },
+            title: "Create User Store"
+        },
+        edit: {
+            configurations: {
+                attributes: {
+                    custom: {
+                        emptyPlaceholder: {
+                            description: "There are no custom attributes created in the system.",
+                            heading: "No Custom Attributes"
+                        },
+                        heading: "Custom Attributes"
+                    },
+                    heading: "Attribute Mappings",
+                    local: {
+                        heading: "Local Attributes"
+                    }
+                },
+                groupAttributes: {
+                    heading: "Group attributes"
+                },
+                userAttributes: {
+                    heading: "User attributes"
+                }
+            },
+            guide: {
+                heading: "Connect the remote user store",
+                steps: {
+                    attributeMapping: {
+                        description:
+                            "Update the <1>attribute mappings</1> according to remote user store that you connected. Make sure to review the mapped attributes otherwise it may cause errors in the user listings.",
```

Comment:
```suggestion
                            "Update the <1>attribute mappings</1> to match the connected remote user store. Please review the mapped attributes carefully to avoid errors when retrieving users from the user store.",
```